modules/nextflow/src/main/groovy/nextflow/Const.groovy

```diff
@@ -50,17 +50,17 @@ class Const {
     /**
      * The application version
      */
-    static public final String APP_VER = "19.1.0-edge"
+    static public final String APP_VER = "19.1.0-SNAPSHOT"
 
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1545514050311
+    static public final long APP_TIMESTAMP = 1546135495890
 
     /**
      * The app build number
      */
-    static public final int APP_BUILDNUM = 5048
+    static public final int APP_BUILDNUM = 5050
 
 
```

Comment:
As before, These numbers are automatically updates, do not add this file to the commit.